asminc/apple2.inc

```diff
@@ -91,3 +91,8 @@ IOUDISOFF   :=  $C07F   ; Enable IOU
 
 ; Control Your Apple
 CYAREG  :=      $C036   ; Bits 0-3=disk detect 4=shadow all banks 7=fast
+
+; PAL/NTSC
+TV_NTSC  :=     0
```

Comment:
:= is only for addresses. For non-address concstants = is to be used. See https://github.com/cc65/cc65/blob/master/asminc/c64.inc for and example getting this right.